newIDE/app/src/ProjectManager/index.js

```diff
@@ -299,6 +306,18 @@ export default class ProjectManager extends React.Component<Props, State> {
     }
   }
 
+  _onEditLayoutVariable = (open: boolean, name: string) => {
+    this.setState({ layoutVariablesDialogOpen: open, layoutName: name });
```

Comment:
You could use a single variable for both, if you make the string nullable :)

Reply:
Ok @4ian . So you suggesting I use a single variable to represent the layoutVariablesDialogOpen and layoutName?

Reply:
Yep